src/main/java/com/denizenscript/depenizen/bukkit/properties/mythicmobs/MythicMobsWorldProperties.java

```diff
@@ -0,0 +1,39 @@
+package com.denizenscript.depenizen.bukkit.properties.mythicmobs;
+
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import com.denizenscript.depenizen.bukkit.bridges.MythicMobsBridge;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.depenizen.bukkit.objects.mythicmobs.MythicMobsMobTag;
+
+public class MythicMobsWorldProperties implements Property {
+    @Override
+    public String getPropertyString() {
+        return null;
+    }
+
+    @Override
+    public String getPropertyId() {
+        return "MythicMobsWorld";
+    }
+
+    @Override
+    public void adjust(Mechanism mechanism) {
+
```

Comment:
newline shouldn't be here, and `handledMechs` is missing... also describes/getFrom. Also this property isn't registered. Clearly you didn't test this at all.

Reply:
yeah didnt mean to include the world .java